Diffusion-macOS/ControlsView.swift

```diff
@@ -155,9 +155,15 @@ struct ControlsView: View {
         VStack {
             Spacer()
             PromptTextField(text: $generation.positivePrompt, isPositivePrompt: true, model: $model)
+                .onChange(of: generation.positivePrompt) { prompt in
+                    Settings.shared.prompt = prompt
+                }
```

Comment:
Interesting, do you think it is a frequent use case to preserve the prompts?

Reply:
Definitely useful for development between builds, if you'd prefer to not keep it around in prod we can add a #DEBUG flag, lmk.

Reply:
No need to use a debug flag, let's keep it as in your PR.